docs/tre-admins/setup-instructions/cicd-pre-deployment-steps.md

```diff
@@ -0,0 +1,134 @@
+# Pre-deployment steps
+
+!!! info
+    See [Environment variables](../environment-variables.md) for full details of the deployment related variables.
+
+## Setup Github Environment
```

Comment:
Environments are only available on paid GitHub plans. See https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment

I'm using a free plan without environments successfully, but steps are slightly different.

Reply:
Since we need an account that can test that, and also it might take some time to setup everything with that, , created a separate story to address that [https://github.com/microsoft/AzureTRE/issues/2790](https://github.com/microsoft/AzureTRE/issues/2790)